store-api-server/src/nft/service/nft.service.ts

```diff
@@ -341,13 +341,40 @@ WHERE idx_address = $1
 `,
       [address],
     );
-    return Object.keys(PHASE2_TOKEN_ID).reduce(
-      (acc, el) => {
-        acc[el] = parseInt(keys.rows.find((r: Record<string, string>) => parseInt(r.token_id, 10) === PHASE2_TOKEN_ID[el])?.count ?? '0', 10);
-        return acc;
-      },
-      {} as Record<string, number>,
+
+    const getKeysDataRawResult = await this.conn.query(
+      `
+SELECT
+  claims.token_id AS token_id,
+  claims.token_type AS token_type,
+  mint.command->'args'->>'metadata_ipfs' AS metadata_ipfs,
+  mint.included_in AS mint_hash
+FROM claims
```

Comment:
This wont take into account onchain transfers. I think this query should be based on what's in `token_gate."storage.ledger_live"` instead.